inc/class-statify-schema.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Statify: Statify_Schema class
+ *
+ * This file contains class for the plugin's DB scheme handling.
+ *
+ * @package   Statify
+ * @since     2.0.0
+ */
+
+// Quit if accessed outside WP context.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Statify DB Schema
+ *
+ * @since 2.0.0
+ */
+class Statify_Schema {
+	/**
+	 * Database tables
+	 *
+	 * @var string[]
+	 */
+	public static $tables = array(
```

Comment:
Maybe this should not be public and/or constant?

Reply:
Const array is PHP 5.6+ iirc (currently supported 5.3+)
But feel free to raise the requirements, if that make things easier here.